src/Headers/ContentSecurityPolicy/Nonce.php

```diff
@@ -0,0 +1,25 @@
+<?php declare(strict_types = 1);
+
+namespace Pd\SecurityHeaders\Headers\ContentSecurityPolicy;
+
+final class Nonce implements IValue
+{
+
+	/**
+	 * @var string
+	 */
+	private $nonce;
+
+
+	public function __construct(string $nonce)
+	{
+		$this->nonce = $nonce;
```

Comment:
Nemusí `nonce` být v nějakém určitém formátu? Pokud ano, tak se tady taky případně dala udělat validace.